src/licensedcode/data/rules/copyheart_syntax_highlighter.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - copyheart
```

Comment:
Add a new line

Reply:
And where is the `copyheart_syntax_highlighter.RULE` rule file?

Reply:
Didn't get what you meant by add a new line